src/App.js

```diff
@@ -1,8 +1,18 @@
+import { Routes, Route } from 'react-router-dom';
+import Books from './components/Books';
+import Categories from './components/Author';
+import Nav from './components/Nav';
+import './App.css';
+
 function App() {
   return (
-    <div className="App">
-      Hello
-    </div>
+    <>
+      <Nav />
+      <Routes>
+        <Route path="/" element={<Books />} />
+        <Route path="/author" element={<Categories />} />
```

Comment:
you did well with your project, kindly note that the second route should be ` /categories ` and not `/author` as stated in the requirement 😄 

![image](https://github.com/Ochiengsteven/bookstore/assets/20537712/47c918a2-625e-4db6-a853-e47db37050b7)